packages/fxa-auth-server/lib/senders/emails/templates/postVerify/index.mjml

```diff
@@ -0,0 +1,25 @@
+<mj-include path="./css/global.css" type="css" css-inline="inline" />
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-sub-body">
+      <span data-l10n-id="postVerify-sub-title">Firefox Account verified. You're almost there.</span>
+    </mj-text>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="postVerify-title">Next sync between your devices!</span>
+    </mj-text>
+    <mj-image css-class="sync-logo" alt="Devices" src="https://accounts-static.cdn.mozilla.net/other/graphic-laptop-mobile.png"></mj-image>
```

Comment:
Ah you know, we probably need a follow up issue for localizing alt text, I don't think we did that at all in current/previous emails (we just put `alt=""`) so we can probably do that after we've converted the remaining templates. We may have questions on what that alt text should be as we continue to build these out, so maybe the task can be an all-encompassing "make sure all images have alt text and that the alt text is localized" kind of ticket.

Reply:
👍 https://mozilla-hub.atlassian.net/browse/FXA-3924